makefiletable.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+
+# Takes a remote file manifest JSON file and creates a Markdown table
+# https://github.com/fair-research/bdbag/blob/master/doc/config.md#remote-file-manifest
+
```

Comment:
@rpwagner you run these 2 scripts manually on your machine to generate the markdown files, right?

Reply:
but the output markdown files are on Globus, not on Github

Reply:
I'll write up a Markdown file describing the process. It was somewhat messy and ad hoc for these first datasets.